### PR TITLE
Prevents loading of the web component script more than once :)

### PR DESCRIPTION
**Pull request by 64BitAsura**

### CODE REVIEW

File: gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java [hunk 1]
```diff
@@ -22,6 +22,9 @@
 import com.google.gwt.core.client.ScriptInjector;
 
 public class GwtPolymer {
+
```
Comment (christiangoudreau) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java]: rm 25

File: gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java [hunk 2]
```diff
@@ -35,9 +38,14 @@ public static void init(LoadCallback callbacks) {
     }
 
     private static void injectWebComponentsScript(final LoadCallback callback) {
+
```
Comment (christiangoudreau) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java]: rm 41

**Comment by christiangoudreau** [comment]
LGTM, @meriouma ?

File: gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java [hunk 3]
```diff
@@ -22,6 +22,8 @@
 import com.google.gwt.core.client.ScriptInjector;
 
 public class GwtPolymer {
+    private static boolean webCompScriptInj;
```
Comment (meriouma) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java]: rename to `webComponentsScriptInjected`

Comment (meriouma) [inline gwt-polymer/src/main/java/com/arcbees/gwtpolymer/base/GwtPolymer.java]: and move this field declaration after the `LoadCallback` definition

**Comment by meriouma** [comment]
Sorry about the delay for code review. just some code style issues, otherwise LGTM

**Comment by christiangoudreau** [comment]
@sambathl Once you're done, we're all good to merge :D

**Comment by olafleur** [comment]
@sambathl You only have the conflicts left to fix. If you want help with that, tell us, and we'll be glad to help you. ;)

**Pull request merged**